odxtools/database.py

```diff
@@ -84,7 +84,7 @@ def get_lookups(layer: Union[DiagLayerContainer, DiagLayer], docnames = None) ->
             for key, value in dict(lookup).items():
                 # In case an ID-REF uses a DOCREF
                 for docname in docnames:
-                    result[docname + ">" + key] = value
+                    result[f"{key}_from_{docname}"] = value
```

Comment:
Line 100 needs to be updated as well

```
docref_lookup = {k: v for k, v in self.id_lookup.items() if '>' in k}
```

Reply:
ok. I suppose we should change the DOCREF implementation a bit in the future: I propose a hashable `OdxReference` class that has a total ordering which includes the IDREF and the optional DOCREF instead of doing references using strings. not now, though...

Reply:
A named tuple would do the job